packages/avro-ts/test/integration.ts

```diff
@@ -0,0 +1,180 @@
+import { User as ComplexRecord } from './__generated__/ComplexRecord.avsc';
+import {
+  UkCoBoostpowerSupportKafkaMessages as AccountNames,
```

Comment:
Biggest change would be namespaces are converted to unique names, as opposed to just `Names` before